Makefile

```diff
@@ -42,7 +44,7 @@ else
 endif
 OFFICIAL_DEVEL_IMAGE := tensorflow/tensorflow:$(TF_VERSION)-devel$(GPU_POSTFIX)
 OFFICIAL_DEVEL_IMAGES := tensorflow/tensorflow:*-devel$(GPU_POSTFIX)
-CPP_IMAGE := rwthika/tensorflow-cc:$(TF_VERSION)$(GPU_POSTFIX)
+CPP_IMAGE := rwthika/tensorflow-cc:$(TF_VERSION)$(GPU_POSTFIX)*
```

Comment:
Are we sure this change does what it is supposed to do?

Reply:
Adjusted in [c9b976a](https://github.com/ika-rwth-aachen/libtensorflow_cc/pull/2/commits/c9b976ac9715eccd3b07c0b4ec2c71e1b106208d).